site/app/controllers/superuser/SuperuserEmailController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace app\controllers\superuser;
+
+use app\libraries\Core;
+use app\libraries\response\WebResponse;
+use app\libraries\response\MultiResponse;
+use app\libraries\response\JsonResponse;
+use app\controllers\AbstractController;
+use app\libraries\routers\AccessControl;
+use app\libraries\User;
+use app\views\superuser\SuperuserEmailView;
+use Symfony\Component\Routing\Annotation\Route;
+use app\models\SuperuserEmail;
+
+/**
+ * @AccessControl(level="SUPERUSER")
+ */
+class SuperuserEmailController extends AbstractController {
+
+    public function __construct(Core $core) {
+        parent::__construct($core);
+    }
+
+    /**
+     * @Route("/superuser/email")
+     * @return MultiResponse
+     */
+    public function showEmailPage(): MultiResponse {
+        return MultiResponse::webOnlyResponse(
+            new WebResponse(SuperuserEmailView::class, 'showEmailPage')
+        );
+    }
+    /**
+     * @Route("/superuser/email/send", methods={"POST"})
+     * @return JsonResponse
+     */
+    public function sendEmail(): JsonResponse {
+        if (!isset($_POST['emailContent']) || $_POST['emailContent'] == '') {
```

Comment:
Same as variables, POST keys (and the HMTL names in form) should be `snake_case`: `$_POST['email_content']`